### PR TITLE
updating __all__ events

### PR DESCRIPTION
**Pull request by phileinsophos**

### CODE REVIEW

File: ppb/events.py [hunk 1]
```diff
@@ -31,18 +31,24 @@ def on_update(self, event: Update, signal):
 
 
 __all__ = (
-    'StartScene',
+    'AssetLoaded',
+    'ButtonPressed',
+    'ButtonReleased',
+    'KeyPressed',
```
Comment (pathunstrom) [inline ppb/events.py:37]: Can we make sure to add KeyReleased to this as well?

**Comment by pathunstrom** [comment]
This looks good except for the one event I asked for, and add yourself to the contributors file.

File: ppb/events.py (hunk 1 above)
Comment (phileinsophos) [reply ppb/events.py:37]: changes done

**Review by pathunstrom — APPROVED** [review]
bors r+

Thanks for the contribution!

**Comment by pathunstrom** [comment]
bors retry

**Comment by pathunstrom** [comment]
bors retry

**Pull request merged**